practices/testing.md

```diff
@@ -374,32 +387,27 @@ Don't.
   config.before(:each) do |example|
     Capybara.current_driver = :selenium if example.metadata[:selenium]
   end
-
   ```
 
 
 ## Factories
 
-- Always include the syntax helpers.
+- Always include the syntax helpers in your tests. This makes `build` and `create` available in your test cases.
 
   ```ruby
   # rails_helper.rb
   config.include FactoryGirl::Syntax::Methods
-
   ```
 
-- Prefer `build` over `create`, if you can get away with it.
+- Prefer `build` over `create`, if you can get away with it. This can lead to faster tests.
 
 - Avoid `build_stubbed`.
 
-- Never hard code emails and similar fields that must be unique. Use `sequence`.
+- Never hard-code emails and similar fields that must be unique. Use `sequence`.
 
   ```ruby
   factory :user do
-    sequence :email do |index|
-      "user-#{index}@email.com"
-    end
-
+    sequence(:email) { |i| "user-#{i}@email.com" }
```

Comment:
We should probably mention na you can put all your sequences in a separate file to avoid duplication.